1.9/administering-clusters/deploying-a-local-dcos-universe.md

```diff
@@ -200,8 +200,25 @@ You can install and run DC/OS services on a datacenter without internet access w
     sudo curl -o /etc/docker/certs.d/master.mesos:5000/ca.crt http://master.mesos:8082/certs/domain.crt
     sudo systemctl restart docker
     ```
-     
-1.  Close the SSH session by typing `exit` or open a new terminal prompt tab. Repeat steps 28 and 29 on each agent node.
+
+1. Configure the Apache Mesos fetcher to trust the downloaded Docker certificate.
+
+   1. Copy the certificate:
+   ```
+   sudo cp /etc/docker/certs.d/master.mesos:5000/ca.crt /var/lib/dcos/pki/tls/certs/ca.crt
+   ```
+   1. Generate a hash:
+   ```
+   cd /var/lib/dcos/pki/tls/certs/
+   openssl x509 -hash -noout -in ca.crt
+   ```
+   1. Create a soft link:
```

Comment:
This is technically called a 'symbolic link' although 'soft link' makes sense given that the alternative link is called a 'hard link'.